packages/back-end/src/api/experiments/putExperiment.ts

```diff
@@ -0,0 +1,34 @@
+import { ExperimentInterface } from "../../../types/experiment";
+import { PutExperimentResponse } from "../../../types/openapi";
+import { createExperiment } from "../../models/ExperimentModel";
+import { toExperimentApiInterface } from "../../services/experiments";
+import { createApiRequestHandler } from "../../util/handler";
+import { putExperimentValidator } from "../../validators/openapi";
+
+export const putExperiment = createApiRequestHandler(putExperimentValidator)(
```

Comment:
It seems like this is for creating new experiments.  For that, we usually use `POST` instead of `PUT` requests.

Reply:
Let me add both then, was thinking `PUT` can also do creation if the entity does not exist, per REST rules.

Reply:
Updated the logic to only `PUT` an existing experiment.